{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_allure.py:19: Exception"}, "start": 1612412064594, "stop": 1612412064594, "uuid": "8b18ed70-c7b8-40bc-91b9-4f672b2550a5", "historyId": "985c3d972d231f87d0914b89f78754fd", "testCaseId": "cb41bd139e929d9eef92825c451cc27a", "fullName": "test_allure#test_broken", "labels": [{"name": "suite", "value": "test_allure"}, {"name": "host", "value": "carolineyuandeMacBook-Pro.local"}, {"name": "thread", "value": "94837-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allure"}]}